demo/tensorflow/app.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (C) 2021-2022, Mindee.
```

Comment:
@odulcy-mindee remove this file ? :)

Reply:
@felixdittrich92 My bad, I forgot it during the rebase